starknet_forge_template/scripts/deploy/Scarb.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "deploy"
+version = "0.1.0"
+
+[dependencies]
+starknet = "2.8.0"
+
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.31.0" }
+sncast_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.32.0" }
+
+[[target.starknet-contract]]
+sierra = true
+casm = true
+
+[tool.snforge]
+exit_first = true
```

there is a missing new line 
```suggestion
exit_first = true

```